ckanext/dataset_subscriptions/templates/dataset-subscriptions_whatsapp_body.j2

```diff
@@ -5,8 +5,9 @@
 {% set group_name = activity.dataset_groups.title -%}
 {% set program_area = activity.dataset_extras.value -%}
 {% set counter = loop.index -%}
-{{ counter|string + ". " + group_name + program_area + activity.data.package.title }}
-{% set datasets_left = num_of_datasets - loop.index -%}
+{{ "*"counter|string + ". " + group_name + program_area + activity.data.package.title"*" }}
```

Comment:
` "*"counter|string` this looks like something which won't work. Do I miss something? A missing "+"?

Reply:
Thanks, you're right